src/WebJobs.Script.WebHost/EncyptedHostAssignmentContextReader.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
```

Comment:
nit: Move these new types to a folder out of the project root, combine them.

Reply:
moved to a new folder ContainerManagement